Cargo.toml

```diff
@@ -2,5 +2,6 @@
 
 members = [
     "fvm_dispatch",
+    "fvm_dispatch_helper",
```

Comment:
`fvm_dispatch_tools`? Helper is rather generic. There's quite likely more functionality we can add to tools for use at build time, such as detecting collisions.